lasagne/nonlinearities.py

```diff
@@ -60,6 +60,87 @@ def tanh(x):
     return theano.tensor.tanh(x)
 
 
+# scaled tanh
+class ScaledTanH(object):
+    """Scaled tanh :math:`\\varphi(x) = \\tanh(\\alpha \\cdot x) \\cdot \\beta`
+
+    This is a modified tanh function which allows to rescale both the input and
+    the output of the activation.
+
+    Scaling the input down will result in decreasing the maximum slope of the
+    tanh and as a result it will be in the linear regime in a larger interval
+    of the input space. Scaling the input up will increase the maximum slope
+    of the tanh and thus bring it closer to a step function.
+
+    Scaling the output changes the output interval to :math:`[-\\beta,\\beta]`.
+
+    Parameters
+    ----------
+    scale_in : float32
```

Comment:
I guess that was copy/pasted from the other docstrings here written by Martin Thoma. At some point, we need to figure out how to call different things and unify all the docstrings in the library accordingly. That's outside the scope of this PR.